client/components/forms/form-setting-explanation/style.scss

```diff
@@ -9,4 +9,17 @@
 	&.is-indented {
 		margin-left: 24px;
 	}
+
+	button.is-borderless {
+		color: $blue-wordpress;
+		font-size: inherit;
+		font-style: inherit;
+		font-weight: inherit;
+		line-height: inherit;
+		padding: 0;
+	}
```

Comment:
Are all these style OK in the shared component or should they be in the specific styles?

Reply:
These are pretty specific to buttons that are children of `FormSettingExplanation`, so I chose to keep them here.